Automattic/jetpack
# Forms: Update package with latest contact-form changes

Part of #28405

## Proposed changes:

* Updates package with latest contact-form changes from trunk

### Other information:


## Jetpack product discussion
pcsBup-N3-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:

Enable the Forms package on your jetpack install by making sure the filter below is added before the contact form module is loaded:

```
add_filter( 'jetpack_contact_form_use_package', '__return_true' );
```

Make sure the package is installed with `jetpack install packages/forms && jetpack install plugins/jetpack`, and build the JS bundle with `jetpack build packages/forms`. 
You can also test the watch functionality with `jetpack watch packages/forms`.

Play with the Form block and check if it works as expected.
Make sure to test the latest style changes: #28212 - #27837
